samples/e2e/EventHubsCaptureEventGridDemo/Readme.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Readme
+description: An overview, how Event Hubs Capture integrates with Event Grid  
+services: event-hubs
+author: shubhavijayasarathy
+
+---
+One of the key scenarios for modern cloud scale apps is seamless integration and notification amongst other apps and services. In this blog post, we introduced [Azure EventGrid](https://azure.microsoft.com/blog/introducing-azure-event-grid-an-event-service-for-modern-applications/) (in public preview), a service designed just for that!
+Today, we will go over a realistic scenario of capturing Azure EventHub data into a SQL Database Warehouse and demonstrate the power and simplicity of using [Azure EventGrid](https://docs.microsoft.com/azure/event-grid/overview) to achieve this.
+
+So, fasten your seat belts…
+*	First, we turn on Capture on Azure Event Hub with Azure blob storage as destination. This generates Azure storage blobs containing EventHub data in Avro format.
+*	Next, we create an Azure EventGrid subscription with source as the Azure EventHub namespace and the destination as an Azure WebJob Function.
```

Comment:
You might want to review this document and make sure the product names are correct. It's "Event Hubs" for the service, and "event hub" for references to an individual hub.